dspace/config/spring/api/discovery.xml

```diff
@@ -57,10 +57,10 @@
                 <entry key="undiscoverable" value-ref="unDiscoverableItems" />
                 <entry key="publication" value-ref="publication"/>
                 <entry key="person" value-ref="person"/>
-                <entry key="organization" value-ref="organization"/>
-                <entry key="publicationIssue" value-ref="publicationIssue"/>
-                <entry key="publicationVolume" value-ref="publicationVolume"/>
-                <entry key="periodical" value-ref="periodical"/>
+                <entry key="orgunit" value-ref="orgUnit"/>
+                <entry key="journalissue" value-ref="journalIssue"/>
+                <entry key="journalvolume" value-ref="journalVolume"/>
+                <entry key="journal" value-ref="journal"/>
                 <entry key="project" value-ref="project"/>
                 <!-- search for an entity that can be a Person or an OrgUnit -->
                 <entry key="personOrOrganization" value-ref="personOrOrganization"/>
```

Comment:
@benbosman, for coherence, do you also propose to rename this setting (and the references for it) from:
`<entry key="personOrOrganization" value-ref="personOrOrganization"/>`
to 
`<entry key="personOrOrgUnit" value-ref="personOrOrgUnit"/>`

Reply:
I've updated this in the PR